configuration/analyzer_config.json

```diff
@@ -1246,7 +1246,7 @@
       "api_key_name": {
         "env_var_key": "GREYNOISE_API_KEY",
         "description": "",
-        "required": false
+        "required": true
```

Comment:
does this become mandatory for Community APIs?

Reply:
I read "Available to unauthenticated users with a limited number of lookups per day" in the documentation so this one should be kept as `false`

Reply:
The experience with the API is better since the limits are higher, and it is preferred for integrations.  The API key is still totally free and takes only a second to generate, so it is always our desire that using an API key be required.

Reply:
I completely understand this but, on the other hand, this limits the chance for the users who just use a tool like IntelOwl for educational purposes. I mean, if you want to have the users to always set an API key it makes completely sense to me but that restriction should be made on your side, not in IntelOwl. So, still, this should be kept as `false` if your service still provide open access. If you reconsider your policy, I'll change this.

Reply:
Happy to change this back, no worries.  I'll include it in my next update.